requirements/base.txt

```diff
@@ -1,4 +1,5 @@
 click >= 8, < 9
+geographiclib >= 2, < 3
```

Comment:
Because we lifted that code from [here](https://github.com/maurycyp/vincenty) which has issues raised by people that have gone ignored. It also loops to convergance based on a max iteration period which we don't know if it will ever fail on because we can't realistically test all edge cases.

geographiclib has no upstream dependencies, so it seems like a safe (and fast) way to have the same functionality that we outsource maintenance to.